src/TurbulenceClosures/isopycnal_rotation_tensor_components.jl

```diff
@@ -1,32 +1,59 @@
 # tracer components of the Redi rotation tensor
 
-@inline function isopycnal_rotation_tensor_xz_fcc(i, j, k, grid::AbstractGrid{FT}, buoyancy, tracers) where FT
+"""
+    AbstractTurbulenceClosure
+
+Abstract supertype for turbulence closures.
+"""
+abstract type AbstractIcopycnalModel end
```

Comment:
Typo --- should be `AbstractIsopycnalModel`

Reply:
I had fixed this **and** pushed! Weird!